docs/src/features/index.md

```diff
@@ -2,4 +2,75 @@
 
 🚧 in Progress
 
-Für jeden Service gibt es eine separate Beschreibung zu dessen Aufgaben und Funktionen.
\ No newline at end of file
+Für jeden Service gibt es eine separate Beschreibung zu dessen Aufgaben und Funktionen.
+
+## Fachliches Datenmodell
+
+Die Relationen zwischen den fachlichen Elementen können sich von Wahl zu Wahl unterscheiden.
```

Comment:
Kannst du mir das bitte erklären?

Reply:
Nehmen wir die Elemente "Stimmzettelgebiet" und "Wahl". Im Fall einer "Europawahl" würde die Relation `Wahl` 1 - 1 `Stimmzettelgebiet` genügen. Im Fall BTW oder LTW ist eine `Wahl` 1- 1..N `Stimmzettelgebiet` rechtfertigt. (Sinkt die Bevölkerung Münchens auf unter 125000 Einwohner haben wir nur ein Stimmkreis, wächst weiter werden wir die aktuellen 9 auch hochschrauben). Das Modell muss natürlich allen Situationen/Wahlen gerecht werden. Möchte damit verdeutlichen, dass wenn man beim Lesen gerade im Kopf eine EUW hat sich nicht wunder sollte warum das Modell mehr umfasst, sondern sich bewusst sein, dass in anderen Wahlen es anders sein kann.

Reply:
Da ein 1..N auch ein mögliches N mit 1 inkludiert würde ich den Satz lieber entfernen. Er sorgt aus meiner Sicht für mehr Verwierrung da er etwas die Korrektheit des Datenmodells infrage stellt.

Reply:
Erledigt hier 22810bc8238623453f05f6fedc21949ec4e8051c